### DISCUSSION TITLE
Only one face is detected (Node)

### DISCUSSION THREAD

**Discussion by Le0X8** [vladmandic/human#342] [Q&A] [answered]
Hi,

I've scanned some pictures with Human, but only one face per picture is detected by the AI. Everytime I tried, with lots of pictures.
It's also not random which face will be chosen, and it not depends on the visibility/quality of the face.

I loaded a picture of my crush and her friends, but only my crush's face will be detected. She wasn't even centered. (Maybe the AI chooses by beauty 😃 )

Is this an actual bug or can I do something in the config?

@Le0X8

**Comment by vladmandic** [comment, marked as answer]
you need to set `config.face.detector.maxDetected = number`.
default is 1 for performance reasons as analyzing any additional detected face does have performance implications and single face is a most common use-case.
